src/main/webapp/WEB-INF/tags/instructor/home/courseTable.tag

```diff
@@ -48,7 +48,7 @@
                     ${sessionRow.status}
                 </span>
             </td>
-            <td class="session-response-for-test<c:if test="${sessionRow.recent}"> recent</c:if>">
```

Comment:
Check for the variable `isRecent` at `InstructorHomeFeedbackSessionRow`. If its sole usage is here (which is removed), then remove the variable as well.